ddtrace/internal/sampling.py

```diff
@@ -180,3 +189,65 @@ def apply_span_sampling_tags(self, span):
         span.set_metric(_SINGLE_SPAN_SAMPLING_RATE, self._sample_rate)
         if self._max_per_second:
             span.set_metric(_SINGLE_SPAN_SAMPLING_MAX_PER_SEC, self._max_per_second)
+
+
+def make_span_sampling_rules():
```

Comment:
```suggestion
def get_span_sampling_rules():
```

not sure I have a strong opinion over which verb is used in the name